src/App.tsx

```diff
@@ -4,29 +4,37 @@ import header from './container/Header';
 import './App.css';
 import store from './redux/index';
 import Error404 from './container/Error404';
+
+import MainLayout from './components/layouts/MainLayout';
+import './assets/css/styles.css';
+import './assets/css/featured.css';
+
 import { Provider as StoreProvider } from 'react-redux';
 import { createGlobalStyle } from 'styled-components';
 import Signup from './pages/user/Signup';
-import Homepage from './container/Homepage';
+import Homepage from './pages/Home';
 import VerifyEmail from './container/VerifyEmail';
 import Login from './pages/user/Login';
 
 const App = () => {
-  return (
-    <StoreProvider store={store}>
-      <Router>
-        <GlobalStyle />
-        <Switch>
-          <Route exact path='/' component={header} />
-          <Route exact path='/homepage' component={Homepage} />
-          <Route exact path='/signup' component={Signup} />
-          <Route exact path='/login' component={Login} />
-          <Route exact path='/success' component={VerifyEmail} />
-          <Route path='*' component={Error404} />
-        </Switch>
-      </Router>
-    </StoreProvider>
-  );
+	return (
+		<StoreProvider store={store}>
+			<Router>
+				<GlobalStyle />
+				<Switch>
+					{/* <Route exact path="/" component={header} /> */}
+					<MainLayout>
```

Comment:
kindly refactor this this line and remove the mainlayout as this page is only for routes